docs/pages/versions/unversioned/introduction/web.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Expo for Web
+sidebar_title: Web Support
+---
+
+If you build your native mobile app with Expo then you can also run it directly in the browser with the CLI and universal Expo SDK. The web part of your app runs with [React Native for web](https://github.com/necolas/react-native-web) which powers massive websites and progressive web apps like [Twitter](https://mobile.twitter.com/), and [Major League Soccer](https://matchcenter.mlssoccer.com/). The Expo SDK also utilizes native browser functionality like Video, Camera, and Gestures without the need for a custom native browser.
+
+> 🚨 Web support has been available in beta since SDK 33, if you find a bug please [report it here](https://github.com/expo/expo/issues) with the `[web]` tag in the title.
+
+## 🔎 How does it work
+
+You get a highly performant React website with progressive web app features enable from the start. Because there is no "magical" canvas work going on in the background, you get a fully accessible website with a great lighthouse score right from the start.
```

Comment:
```suggestion
You get a highly performant React website with progressive web app features enabled from the start. Because there is no "magical" canvas work going on in the background, you get a fully accessible website with a great lighthouse score right from the start.
```